### PR TITLE
Optimize: truncate parameterString

### PR DESCRIPTION
**Pull request by novayoung**
cut the string as soon as it reached the length limitation

Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [* ] Improve performance

- Related issues
https://github.com/apache/skywalking/issues/5383
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
cut the string as soon as it reached the length limitation

### CODE REVIEW

File: apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/PreparedStatementParameterBuilder.java [hunk 1]
```diff
@@ -58,6 +57,11 @@ private String getParameterString() {
             }
             stringBuilder.append(parameter);
             first = false;
+            
+            //  cut the string as soon as it reached the length limitation
+            if (maxLength > 0 && stringBuilder.length() > maxLength){
+                return String.format("[%s]", stringBuilder.toString().substring(0, maxLength) + "...");
```
Comment (dmsolr) [inline apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/PreparedStatementParameterBuilder.java:63]: Why don't you place `...` in `[%s.]` like `[%s....]`.  :-)

Comment (novayoung) [reply apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/PreparedStatementParameterBuilder.java:63]: > Why don't you place `...` in `[%s]` like `[%s....]`. :-)

nice

**Review by wu-sheng — COMMENTED** [review]
Have you followed the code style? All CI tests fail. Please fix.

**Comment by novayoung** [comment]
> Have you followed the code style? All CI tests fail. Please fix.

OK

**Comment by kezhenxu94** [comment]
@novayoung test failed, please fix

**Review by kezhenxu94 — APPROVED** [review]

**Review by dmsolr — APPROVED** [review]

**Pull request merged**